test/SILGen/boxed_existentials.swift

```diff
@@ -91,8 +91,10 @@ func test_property_of_lvalue(_ x: Error) -> String {
 // CHECK:         [[VALUE:%.*]] = open_existential_box [[VALUE_BOX]] : $Error to $*[[VALUE_TYPE:@opened\(.*\) Error]]
 // CHECK:         [[COPY:%.*]] = alloc_stack $[[VALUE_TYPE]]
 // CHECK:         copy_addr [[VALUE]] to [initialization] [[COPY]]
+// CHECK:         [[BORROW:%.*]] = alloc_stack $[[VALUE_TYPE]]
```

Comment:
It's unfortunate that we're copying the value twice now

Reply:
I wasn't sure about that. @gottesmm is it necessary to introduce a copy to `formalAccessBorrow` address-only values?

Reply:
If it’s only for address only values it might be that opaque values work will resolve this on its own

Reply:
Doing a formal access borrow of an address just returns the value. So I don't think it has to do with the formal access borrow itself. I think it comes from not having a cleanup on the address and some code inside the emission copying b/c there isn't a cleanup to make sure it has access to the value. So it is more likely to be in argument emission I think.